R/vdiagram-styles.R

```diff
@@ -1,17 +1,23 @@
 vplot <- function(vdiagramData) {
+  extendStageBy = 0.5
   vplot <- gsplot() %>%
     points(NA,NA, ylab='Stage, in feet', xlab='Shift, in feet') %>%
-    callouts(x=c(0,0),y=c(getMinStage(data, required = TRUE), maxStage), labels="", col = 'red', lwd = 3, angle=0, legend.name="Max and min gage height for the period shown") %>%
+    callouts(x=c(0,0),y=c(vdiagramData$minStage, vdiagramData$maxStage), labels="", 
+             col = 'red', lwd = 3, angle=0, legend.name="Max and min gage height for the period shown") %>%
     grid(lty = "dotted") %>%
     axis(side=c(2,4)) %>%
-    addVdiagErrorBars(x = obsShift, y = obsGage, xError0 = minShift, xError1 = maxShift, histFlag, IDs = obsIDs)
+    addVdiagErrorBars(x = vdiagramData$obsShift, y = vdiagramData$obsGage, 
```

Comment:
For the `addVdiagErrorBars`, there's a gsplot function called `error_bar` that might make this simpler, rather than having arrows + points called.